connect-inject/handler.go

```diff
@@ -175,6 +175,18 @@ type Handler struct {
 	DefaultProxyMemoryRequest resource.Quantity
 	DefaultProxyMemoryLimit   resource.Quantity
 
+	// Resource settings for Init Copy Container
```

Comment:
nit: comments should end with periods as per effective go (this isn't the case throughout the codebase because we're not super strict).